syntax/output.php

```diff
@@ -79,9 +79,13 @@ public function handle($match, $state, $pos, Doku_Handler $handler) {
      * @return bool If rendering was successful.
      */
     public function render($mode, Doku_Renderer $R, $data) {
+        global $ACT;
         global $ID;
         global $INFO;
         global $REV;
+        if (act_clean($ACT) !== 'show') {
+            return true;
+        }
```

Comment:
I'm still unsure if this check is necessary and if it actually could be harmful. What if a different mode triggers xhtml rendering and that is cached then. Like some kind of Ajax plugin or something...